### PR TITLE
Pull cloud url from env var

### PR DESCRIPTION
**Pull request by collindutter**

### CODE REVIEW

**Review by zachgiordano — APPROVED** [review]
🚀

**Review by SavagePencil — DISMISSED** [review]
Open question; will follow up in Slack

File: griptape/drivers/event_listener/griptape_cloud_event_listener_driver.py [hunk 1]
```diff
@@ -28,5 +39,6 @@ def validate_run_id(self, _, run_id: str):
 
     def try_publish_event(self, event: BaseEvent) -> None:
         url = urljoin(self.base_url.strip("/"), f"/api/structure-runs/{self.run_id}/events")
+        print(url)
```
Comment (SavagePencil) [inline griptape/drivers/event_listener/griptape_cloud_event_listener_driver.py:42]: on purpose?

**Review by SavagePencil — APPROVED** [review]
One open Q in there. If deliberate, go ahead and merge.

File: griptape/drivers/event_listener/griptape_cloud_event_listener_driver.py (hunk 1 above)
Comment (collindutter) [reply griptape/drivers/event_listener/griptape_cloud_event_listener_driver.py]: Nope!

**Review by SavagePencil — APPROVED** [review]

**Pull request merged**